test/parallel/test-punycode.js

```diff
@@ -1,153 +1,237 @@
 'use strict';
 require('../common');
-var punycode = require('punycode');
-var assert = require('assert');
+const punycode = require('punycode');
+const assert = require('assert');
 
-assert.equal(punycode.encode('ü'), 'tda');
-assert.equal(punycode.encode('Goethe'), 'Goethe-');
-assert.equal(punycode.encode('Bücher'), 'Bcher-kva');
-assert.equal(punycode.encode(
+assert.strictEqual(punycode.encode('ü'), 'tda');
+assert.strictEqual(punycode.encode('Goethe'), 'Goethe-');
+assert.strictEqual(punycode.encode('Bücher'), 'Bcher-kva');
+assert.strictEqual(punycode.encode(
     'Willst du die Blüthe des frühen, die Früchte des späteren Jahres'),
     'Willst du die Blthe des frhen, die Frchte des spteren Jahres-x9e96lkal');
-assert.equal(punycode.encode('日本語'), 'wgv71a119e');
+assert.strictEqual(punycode.encode('日本語'), 'wgv71a119e');
 
-assert.equal(punycode.decode('tda'), 'ü');
-assert.equal(punycode.decode('Goethe-'), 'Goethe');
-assert.equal(punycode.decode('Bcher-kva'), 'Bücher');
-assert.equal(punycode.decode(
+assert.strictEqual(punycode.decode('tda'), 'ü');
+assert.strictEqual(punycode.decode('Goethe-'), 'Goethe');
+assert.strictEqual(punycode.decode('Bcher-kva'), 'Bücher');
+assert.strictEqual(punycode.decode(
     'Willst du die Blthe des frhen, die Frchte des spteren Jahres-x9e96lkal'),
     'Willst du die Blüthe des frühen, die Früchte des späteren Jahres');
-assert.equal(punycode.decode('wgv71a119e'), '日本語');
+assert.strictEqual(punycode.decode('wgv71a119e'), '日本語');
 
 // http://tools.ietf.org/html/rfc3492#section-7.1
-var tests = {
+const tests = [
   // (A) Arabic (Egyptian)
-  'egbpdaj6bu4bxfgehfvwxn':
-      '\u0644\u064A\u0647\u0645\u0627\u0628\u062A\u0643\u0644\u0645\u0648' +
-      '\u0634\u0639\u0631\u0628\u064A\u061F',
+  {
+    string: 'ليهمابتكلموشعربي؟',
+    encoded: 'egbpdaj6bu4bxfgehfvwxn',
+    decoded: '\u0644\u064A\u0647\u0645\u0627\u0628\u062A\u0643\u0644\u0645' +
```

Comment:
If that's how they appear in the RFC, I'd say maybe leave them just in case? It might also help in debugging, if the test ever starts failing, to have the bytecodes handy.